src/SpacetimeDBClient.cs

```diff
@@ -205,6 +203,97 @@ static DbValue Decode(IRemoteTableHandle table, EncodedValue value, out object?
         private static readonly Status Committed = new Status.Committed(default);
         private static readonly Status OutOfEnergy = new Status.OutOfEnergy(default);
 
+        private static ServerMessage DecompressDecodeMessage(byte[] bytes)
+        {
+            using var stream = new MemoryStream(bytes, 1, bytes.Length - 1);
+
+            switch (bytes[0])
```

Comment:
Is it possible for bytes to be an empty array? If so it'll fail here and in creating the memory stream